packages/core/package.json

```diff
@@ -31,12 +31,12 @@
   "dependencies": {
     "@backstage/config": "^0.1.1-alpha.20",
     "@backstage/core-api": "^0.1.1-alpha.20",
+    "@backstage/components": "^0.1.1-alpha.20",
```

Comment:
so this makes a circular dependency between core and components

Reply:
Yep. Will fix.